### PR TITLE
[YUNIKONR-2179] Extend the coverage of PartitionQueueDAOInfo

### PR DESCRIPTION
**Pull request by brandboat**
### What is this PR for?
Rename method names
Extend the coverage of PartitionQueueDAOInfo

### What type of PR is it?
* [x] - Improvement

### Todos
N/A

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2179

### How should this be tested?
covered by unit tests

### Screenshots (if appropriate)
N/A

### Questions:
N/A

### CODE REVIEW

File: pkg/webservice/handlers_test.go [hunk 1]
```diff
@@ -1544,7 +1595,7 @@ func assertParamsMissing(t *testing.T, resp *MockResponseWriter) {
 	assert.Equal(t, errInfo.StatusCode, http.StatusBadRequest)
 }
 
-func assertPartitionExists(t *testing.T, resp *MockResponseWriter) {
+func assertPartitionNotExists(t *testing.T, resp *MockResponseWriter) {
```
Comment (chia7712) [inline pkg/webservice/handlers_test.go:1598]: there are other similar issues. for example: `assertQueueExists`, `assertApplicationExists`, and `assertNodeIDExists`. Maybe we can fix them at all?

File: pkg/webservice/handlers_test.go [hunk 2]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.Equal(t, child.IsLeaf, true)
+	assert.Equal(t, child.IsManaged, true)
+	assert.Equal(t, child.Parent, "root.a")
+	assert.Equal(t, child.AbsUsedCapacity == nil, true)
+	assert.Equal(t, child.MaxRunningApps, uint64(0))
+	assert.Equal(t, child.RunningApps, uint64(0))
+	assert.Equal(t, child.CurrentPriority, configs.MinPriority)
+	assert.Equal(t, child.AllocatingAcceptedApps == nil, true)
+	assert.Equal(t, len(child.Properties), 1)
+	assert.Equal(t, child.Properties[configs.ApplicationSortPolicy], policies.StateAwarePolicy.String())
```
Comment (chia7712) [inline pkg/webservice/handlers_test.go:1127]: how about adding a test for different policy on child queue?

**Review by chia7712 — COMMENTED** [review]
@brandboat thanks for completing tests. a couple of comments left.

File: pkg/webservice/handlers_test.go [hunk 3]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
-	assert.Equal(t, partitionQueuesDao.Children[2].Parent, "root")
+	// assert root fields
+	assert.Equal(t, partitionQueuesDao.QueueName, configs.RootQueue)
+	assert.Equal(t, partitionQueuesDao.Status, objects.Active.String())
+	assert.Equal(t, partitionQueuesDao.Partition, configs.DefaultPartition)
+	assert.Equal(t, partitionQueuesDao.PendingResource == nil, true)
+	assert.Equal(t, partitionQueuesDao.MaxResource == nil, true)
+	assert.Equal(t, partitionQueuesDao.GuaranteedResource == nil, true)
+	assert.Equal(t, partitionQueuesDao.AllocatedResource == nil, true)
+	assert.Equal(t, partitionQueuesDao.PreemptingResource == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1068]: nit: `assert.Assert()` instead of  `Equal()`

File: pkg/webservice/handlers_test.go [hunk 4]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.Equal(t, partitionQueuesDao.HeadRoom == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1069]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 5]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.Equal(t, partitionQueuesDao.IsLeaf, false)
+	assert.Equal(t, partitionQueuesDao.IsManaged, true)
+	assert.Equal(t, partitionQueuesDao.Parent, "")
+	assert.Equal(t, partitionQueuesDao.AbsUsedCapacity == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1073]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 6]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.Equal(t, partitionQueuesDao.MaxRunningApps, uint64(0))
+	assert.Equal(t, partitionQueuesDao.RunningApps, uint64(0))
+	assert.Equal(t, partitionQueuesDao.CurrentPriority, configs.MinPriority)
+	assert.Equal(t, partitionQueuesDao.AllocatingAcceptedApps == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1077]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 7]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.Equal(t, partitionQueuesDao.Properties[configs.ApplicationSortPolicy], policies.StateAwarePolicy.String())
+	assert.DeepEqual(t, partitionQueuesDao.TemplateInfo, &templateInfo)
+
+	// assert child root.a fields
+	assert.Equal(t, len(partitionQueuesDao.Children), 1)
+	child := &partitionQueuesDao.Children[0]
+	assert.Equal(t, child.QueueName, "root.a")
+	assert.Equal(t, child.Status, objects.Active.String())
+	assert.Equal(t, child.Partition, "")
+	assert.Equal(t, child.PendingResource == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1088]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 8]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.DeepEqual(t, child.MaxResource, maxResource.DAOMap())
+	assert.DeepEqual(t, child.GuaranteedResource, guaranteedResource.DAOMap())
+	assert.Equal(t, child.AllocatedResource == nil, true)
+	assert.Equal(t, child.PreemptingResource == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1092]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 9]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.DeepEqual(t, child.HeadRoom, maxResource.DAOMap())
+	assert.Equal(t, child.IsLeaf, false)
+	assert.Equal(t, child.IsManaged, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1095]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 10]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.Equal(t, child.Parent, configs.RootQueue)
+	assert.Equal(t, child.AbsUsedCapacity == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1097]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 11]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.Equal(t, child.MaxRunningApps, uint64(0))
+	assert.Equal(t, child.RunningApps, uint64(0))
+	assert.Equal(t, child.CurrentPriority, configs.MinPriority)
+	assert.Equal(t, child.AllocatingAcceptedApps == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1101]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 12]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.Equal(t, child.Properties[configs.ApplicationSortPolicy], policies.StateAwarePolicy.String())
+	assert.DeepEqual(t, child.TemplateInfo, &templateInfo)
+
+	// assert child root.a.a1 fields
+	assert.Equal(t, len(partitionQueuesDao.Children[0].Children), 1)
+	child = &partitionQueuesDao.Children[0].Children[0]
+	assert.Equal(t, child.QueueName, "root.a.a1")
+	assert.Equal(t, child.Status, objects.Active.String())
+	assert.Equal(t, child.Partition, "")
+	assert.Equal(t, child.PendingResource == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1112]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 13]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.DeepEqual(t, child.MaxResource, maxResource.DAOMap())
+	assert.DeepEqual(t, child.GuaranteedResource, guaranteedResource.DAOMap())
+	assert.Equal(t, child.AllocatedResource == nil, true)
+	assert.Equal(t, child.PreemptingResource == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1116]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 14]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.DeepEqual(t, child.HeadRoom, maxResource.DAOMap())
+	assert.Equal(t, child.IsLeaf, true)
+	assert.Equal(t, child.IsManaged, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1119]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 15]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.Equal(t, child.Parent, "root.a")
+	assert.Equal(t, child.AbsUsedCapacity == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1121]: nit: `assert.Assert()` instead of `Equal()`

File: pkg/webservice/handlers_test.go [hunk 16]
```diff
@@ -1046,43 +1029,111 @@ func TestGetPartitionQueuesHandler(t *testing.T) {
...
+	assert.Equal(t, child.MaxRunningApps, uint64(0))
+	assert.Equal(t, child.RunningApps, uint64(0))
+	assert.Equal(t, child.CurrentPriority, configs.MinPriority)
+	assert.Equal(t, child.AllocatingAcceptedApps == nil, true)
```
Comment (pbacsko) [inline pkg/webservice/handlers_test.go:1125]: nit: `assert.Assert()` instead of `Equal()`

**Review by pbacsko — CHANGES_REQUESTED** [review]

File: pkg/webservice/handlers_test.go (hunk 16 above)
Comment (brandboat) [reply pkg/webservice/handlers_test.go]: Thanks for suggestion, already address comments in [9621f51](https://github.com/apache/yunikorn-core/pull/772/commits/9621f515c3aaedbbdf58803279ab78ce9d0f9ece), thanks !

File: pkg/webservice/handlers_test.go (hunk 2 above)
Comment (brandboat) [reply pkg/webservice/handlers_test.go]: Already address comments in [563a5ce](https://github.com/apache/yunikorn-core/pull/772/commits/563a5ce666c9d63121033ff2b513ccc3e4241d24), many thanks !

**Review by pbacsko — APPROVED** [review]
+1 LGTM

**Pull request closed without merging**